client/landing/gutenboarding/stores/onboard/index.ts

```diff
@@ -39,6 +39,7 @@ registerStore< State >( STORE_KEY, {
 		'selectedSite',
 		'selectedFeatures',
 		'plan',
+		'randomizedDesigns',
```

Comment:
@andrewserong  One question/comment I forgot yesterday:  There are many area of the code with "lists" of variables or functions (I don't mean arrays, using that word loosely for this comment) that are in a random order, I had to resist the urge to alphabetize.  Ditto on the order of selectors, reducers, actions, etc in their respective files in stores/onboard/selectors.ts.  Ditto with @imports at top of files.  I simply added my `randomizedDesigns` additions to the end of these files.  Or added new imports at the end of the import list or sometimes in the middle if it "made sense".

Alphabetizing seems most logical to me, especially given the number of functions in redux related files.   Figured I bring it up for conversation sake. 🤓

Reply:
> I had to resist the urge to alphabetize.

I have an (almost) irrepressible urge to alphabetize this kind of stuff as well! 🤣 

I mostly hold back if the change is not related to the feature/bug I'm working on. Only for the sake of keeping PRs small.

I see that most folks add vars/props to the end of the 'list'. Maybe because IDE symbol linking or plain ol' Ctrl-F

Feel free to open a janitorial PR and alphabetize a few lists if you can't resist the urge!!! :)

Reply:
Yes, I totally get what you mean with alphabetising things! If it's within the one line (like an import statement), I'll often go ahead and re-order alphabetically, but if it's across multiple lines like this one, separate follow-up janitorial PRs are a great way to go, to keep the readability of your PR in check.

We also _love_ janitorial PRs, they're easy to review and it's quite satisfying seeing the codebase get a little cleaner one PR at a time!

Reply:
Oh great!!  I will open a janitorial PR when I'm done with this PR!! 🥳